concepts/system-model/index.md

```diff
@@ -49,6 +47,42 @@ The principal message primitives of the networking communication layer are:
 Transaction calls execute against local state and are not broadcast to the network, per standard Ethereum.
 :::
 
+### P2P networking protocols
+P2P networking protocols are based on Ethereum devp2p, RLPx as transport to an ethereum wire protocol (eth66). Peer to peer gossiping is separated into two channels: ethereum wire protocol for _transaction_ gossiping, and a dedicated consensus channel for _consensus_ gossiping during Tendermint BFT consensus rounds. These run  on different TCP ports, defaults of `30303` and `20203` respectively.
```

Comment:
I think the first sentence is kinda confusing and doesn't give much insight so I would just remove it.

Reply:
Edited out.

"P2P networking protocols are based on Ethereum devp2p, RLPx as transport to an ethereum wire protocol (eth66)." removed in commit 8a6b4b19931f5c2e144e80c7bfb012c0567b2e08.